src/ftxui/component/screen_interactive.cpp

```diff
@@ -858,6 +858,16 @@ void ScreenInteractive::Draw(Component component) {
   ResetCursorPosition();
   std::cout << ResetPosition(/*clear=*/resized);
 
+  // clear terminal output if screen dimx decreases
+  // only on primary screen
+  // only on POSIX systems (linux/macos)
+#if !defined(_WIN32)
```

Comment:
Is there any reasons to exclude windows?

Reply:
I was not able to test this on windows, so i decided to exclude it. As i can see, windows supports this escape sequences so i will remove that